packages/client/src/components/Categories/CategoriesAndSearchBarAndSortBy.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { CategoryButton } from './CategoryButton';
+import { Box } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { useMoviesAndCategoriesContext } from '../../context/FetchCategoriesAndMovies';
+import { SearchBar } from '../SearchBar/SearchBar';
+import { CustomSelect } from '../Select/CustomSelect';
+
+export const CategoriesAndSearchBarAndSortBy = () => {
+  const { state, dispatch } = useMoviesAndCategoriesContext();
+  const theme = useTheme();
+  const handleOnClick = (catId) => {
+    dispatch({
+      type: 'categoryId',
+      payload: state.categoryId === catId ? null : catId,
+    });
+  };
+  const handleSearchTextChange = (e) => {
+    dispatch({
+      type: 'changeSearchText',
+      payload: { name: e.target.name, value: e.target.value },
+    });
+  };
+
+  const handleSortByOptionChange = (e) => {
+    dispatch({
+      type: 'changeSortBy',
+      payload: {
+        name: e.target.name,
+        value: e.target.value,
+      },
+    });
+  };
+
+  if (state.categoryLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (state.categoryFetchError) {
+    return <div>Error: {state.categoryFetchError.message}</div>;
+  }
+
+  return (
+    <Box
+      sx={{
```

Comment:
On Sunday, we talked extensively about style approach optimization, but here I still see quite a few `sx` overrides with a lot of the same properties. 
Try to revisit the styling question in the team and see what approach you can take. 
You can take inspiration from the Landing team that chose a strategy and is sticking to it quite well. 👍🏽